system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -68,7 +75,8 @@ bool PeriodicMeasurementNode::SetupStart()
     measurement_period_, [this]() {this->PerformPeriodicMeasurement();});
 
   if (publisher_ == nullptr) {
-    publisher_ = create_publisher<MetricsMessage>(publishing_topic_, 10 /*history_depth*/);
+    publisher_ = create_publisher<MetricsMessage>(collector_node_constants::kStatisticsTopicName,
```

Comment:
Oh I see what you're doing now. Why aren't we keeping the member? This PR is actually removing functionality where we could pull the publishing topic out of NodeOptions. The work required seems minimal and I don't see why we can't do it, as a read only parameter, here.

Reply:
> This PR is actually removing functionality where we could pull the publishing topic out of NodeOptions.

I was going to do that, too, but this is what @thomas-moulard suggested under on the same rationale as the node name.

Reply:
I took @thomas-moulard 's comment to mean we don't need to set and test a custom topic name in test. I'm going to follow up with him as we've discussed allowing a user to configure the topic name.

Reply:
This was the discussion in the team chatroom. Topic name will be configured via remapping (`--ros-args --remap ...`).

Reply:
> --ros-args --remap

Got it, found this explanation: https://index.ros.org/doc/ros2/Tutorials/Node-arguments/#example